FAQ.md

```diff
@@ -508,6 +508,47 @@ information_schema 库中的 tables 表里的 create_time 即为表的真实创
 
 TiDB 在执行 SQL 时，预估出来每个 operator 处理了超过 10000 条数据就认为这条 query 是 expensive query。可以通过修改 tidb-server 配置参数来对这个门限值进行调整，调整后需要重新启动 tidb-server。
 
+#### 3.3.10 在 TiDB 中如何控制或改变 SQL 提交的执行优先级？
+
+首先需要了解高优先级和低优先级的语法元素
+
+- HIGH_PRIORITY：该语句为高优先级语句，TiDB 在执行阶段会优先处理这条语句
+- LOW_PRIORITY：该语句为低优先级语句，TiDB 在执行阶段会降低这条语句的优先级
+
+以上两种参数可以结合 TiDB 的 DML 语言进行使用，具体使用方式可以参考[官方文档](sql/dml.md)，使用方法举例如下：
+
+1）通过在数据库中写 SQL 的方式来调整优先级：
+
+```
+select HIGH_PRIORITY | LOW_PRIORITY count(*) from table_name;
+insert HIGH_PRIORITY | LOW_PRIORITY into table_name insert_values;
+delete HIGH_PRIORITY | LOW_PRIORITY from table_name;
+update HIGH_PRIORITY | LOW_PRIORITY table_reference set assignment_list where where_condition;
+replace HIGH_PRIORITY | LOW_PRIORITY into table_name;
+```
+
+2）全表扫会自动调整为低优先级，analyze 也是默认低优先级。
+
+#### 3.3.11 在 TiDB 中 auto analyze 的触发策略是怎样的？
+
+触发策略：新表达到 1000 条，会自动触发。
+
+当表的（修改数/当前总行数）大于 `tidb_auto_analyze_ratio` 的时候，会自动触发 analyze 语句。`tidb_auto_analyze_ratio` 的默认值为 0，即关闭此功能。为了保险起见，在开启此功能的时候，保证了其最小值为 0.3。但是不能大于等于 `pseudo-estimate-ratio`（默认值为 0.7），否则会有一段时间使用 pseudo 统计信息，建议设置值为 0.5。
+
+#### 3.3.12 Select 语句在 TiDB 的 RC 与 RR 隔离级别下的实现过程，以及对应用的影响是怎样的？
+
+TiDB 实现了两种隔离级别：读已提交（Read Committed, RC）和可重复读（Repeatable Read, RR）。
```

Comment:
If the content inside the parentheses is all in English, use the English parentheses and leave a whitespace outside the parentheses.